callback.go

```diff
@@ -0,0 +1,14 @@
+package dcpkafka
+
+import "github.com/Trendyol/go-dcp-kafka/kafka/message"
+
+type callback struct {
```

Comment:
do we need this struct? maybe we can move the Callback(SinkResponseHandler) interface here and remove below logics?